src/parser/jobs/blm/Speedmod.js

```diff
@@ -1,19 +1,63 @@
 import CoreSpeedmod from 'parser/core/modules/Speedmod'
 
 import STATUSES from 'data/STATUSES'
+import {BLM_GAUGE_EVENT} from './Gauge'
+import {FIRE_SPELLS, ICE_SPELLS} from './Elements'
+
+const MAX_ASTRAL_UMBRAL_STACKS = 3
+const FAST_CAST_SCALAR = 0.5
 
 export default class Speedmod extends CoreSpeedmod {
-	/* NOTE: Use this to force modules to run before Speedmod. ie: normalise to generate Huton events so Speedmod can pick them up natively
 	static dependencies = [
 		...CoreSpeedmod.dependencies,
-		'forms',
+		'gauge', // eslint-disable-line xivanalysis/no-unused-dependencies
 	]
-	*/
+
+	_astralFireStacks = 0
+	_umbralIceStacks = 0
+
+	_isFastCast = false
 
 	constructor(...args) {
 		super(...args)
 		this.SPEED_BUFF_STATUS_IDS.push(
 			STATUSES.CIRCLE_OF_POWER.id, // Ley Lines buff
 		)
 	}
+
+	jobSpecificNormaliseLogic(event) {
```

Comment:
Ah, this must be why you need that logic in normalise. Still a bit uncomfortable with that... the more we put in normalise, the more needs to be put in normalise, the more needs... you get the point. Slippery slope and all but I'm concerned we'll end up with no hooks at all soon :blobsweat:.

Not got the brain space to suggest alternative implementations for the whole speedmod stack, but worth discussing modifications to unlock massive chunks of stuff from normalisers.